Diagnostic/watcherutil.py

```diff
@@ -19,8 +19,9 @@
 class Watcher():
     lastModTime = os.path.getmtime('/etc/fstab')
     logger = None
+    consoleStream = None
```

Comment:
I don't see this variable used anywhere else. Did you mean to use this somewhere? If not, should we remove this?